script.js

```diff
@@ -1,4 +1,206 @@
+  // player 1 wins
+  if (diceOrderPlayer1 > diceOrderPlayer2) {
+    playersScore[0] += 1;
+    var output1 = `👱‍♂️ ${players[0]} order is ${diceOrderPlayer1}.<br/>👱‍♂️ ${players[0]} wins 💪.<br>👱‍♂️ ${players[0]} score is score ${playersScore[0]}.<br/>👩‍🦰 ${players[1]} score is ${playersScore[1]}..<br/>This is round ${gameRound} of 10 games 🏓. Game has ended😥🛑🏅.<br/>`;
+    var restartGame = initGame();
+    var myOutputValue = output1 + restartGame;
+  } // player 2 wins
+  if (diceOrderPlayer2 > diceOrderPlayer1) {
+    playersScore[1] += 1;
+    output1 = `👱‍♂️ ${players[0]} order is ${diceOrderPlayer1}.<br/>👩‍🦰 ${players[1]} wins 💪.<br>👱‍♂️ ${players[0]} score is ${playersScore[0]}.<br/>👩‍🦰 ${players[1]} score is ${playersScore[1]}.  <br/>This is round ${gameRound} of 10 games🏓. Game has ended😥🛑📉.<br/>`;
+    restartGame = initGame();
+    myOutputValue = output1 + restartGame;
+  } // draw
+  if (diceOrderPlayer1 == diceOrderPlayer2) {
+    output1 = `👱‍♂️ ${players[0]} order is ${diceOrderPlayer1}. It is a 👔 tie. <br>👱‍♂️ ${players[0]} score is ${playersScore[0]}.<br/>👩‍🦰 ${players[1]} score is ${playersScore[1]}.<br/>  This is round ${gameRound} of 10 games 🏓. Game has ended.😥🛑💢<br/>`;
+    restartGame = initGame();
+    myOutputValue = output1 + restartGame;
+  }
   return myOutputValue;
 };
+
+// function determine win/lose/draw //updates players scores // auto restarts game.
+var whoWins = function () {
+  // converts string type dice order to Number for comparison
+  var diceOrderPlayer1 = Number(diceOrder1);
+  var diceOrderPlayer2 = Number(diceOrder2);
+  gameRound += 1;
+
+  // player 1 wins
+  if (diceOrderPlayer1 > diceOrderPlayer2) {
+    playersScore[0] += 1;
+    var myOutputValue = `👱‍♂️ ${players[0]} order is ${diceOrderPlayer1}.<br/>👱‍♂️ ${players[0]} wins 💪.<br>👱‍♂️ ${players[0]} score is ${playersScore[0]}.<br/>👩‍🦰 ${players[1]} score is ${playersScore[1]}.<br/> Next up is 👱‍♂️ ${players[0]}, please press submit to 🎲roll.<br?>This is round ${gameRound} of 10 games.🏓`;
+  }
+
+  // player 2 wins
+  if (diceOrderPlayer2 > diceOrderPlayer1) {
+    playersScore[1] += 1;
+    myOutputValue = `👱‍♂️ ${players[0]} order is ${diceOrderPlayer1}.<br/>👩‍🦰 ${players[1]} wins 💪.<br>👱‍♂️ ${players[0]} score is ${playersScore[0]}.<br/>👩‍🦰 ${players[1]} score is ${playersScore[1]}. <br/> Next up is 👱‍♂️ ${players[0]}, please press submit to 🎲 roll. <br/>This is round ${gameRound} of 10 games.🏓`;
+  }
+  // draw game
+  if (diceOrderPlayer1 == diceOrderPlayer2) {
+    myOutputValue = `👱‍♂️ ${players[0]} order is ${diceOrderPlayer1}}. It is a 👔 Tie. <br/>👱‍♂️ ${players[0]}, submit to 🎲 roll.<br/> This is ${gameRound} of 10 games🏓.`;
+  }
+  // end, resets, tally game
+  if (gameRound == 10) {
+    myOutputValue = endGameTallyForceRestart(
+      diceOrderPlayer1,
+      diceOrderPlayer2
+    );
+  }
+  return myOutputValue;
+};
+
+// browser output function
+var main = function (input) {
```

Comment:
great job with the helper functions and consistent commenting throughout your code!